canonical-websites/cloud-init.io
# Update to Vanilla 1.6.5 + minor styling fixes to homepage

## Done

- Updated to Vanilla 1.6.5
- Styling fixes/updates

## QA

- `./run`
- Navigate to http://http://0.0.0.0:8009//
- Check that site is now running Vanilla 1.6.5
- Check that there are no styling errors

## Issue / Card

Fixes https://github.com/vanilla-framework/vanilla-framework/issues/1543
